connector/src/main/java/org/jboss/as/connector/subsystems/common/pool/Constants.java

```diff
@@ -169,22 +168,8 @@ public void marshallAsElement(final AttributeDefinition attribute, final ModelNo
             .build();
 
     public static PropertiesAttributeDefinition CAPACITY_DECREMENTER_PROPERTIES = new PropertiesAttributeDefinition.Builder(CAPACITY_DECREMENTER_PROPERTIES_NAME, true)
-            .setXmlName(org.jboss.jca.common.api.metadata.common.Extension.Tag.CONFIG_PROPERTY.getLocalName())
```

Comment:
I would feel more comfortable if there was some coverage of this in the datasources subsystem tests